code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm

```diff
@@ -304,7 +304,7 @@
 /datum/reagent/medicine/charcoal/on_mob_life(mob/living/M)
 	M.adjustToxLoss(-2*REM)
 	for(var/datum/reagent/R in M.reagents.reagent_list)
-		if(R != src)
+		if(R != src && !istype(R, /datum/reagent/toxin/wasting_toxin)) //Changeling death sting toxin can't be removed by charcoal, only delayed
```

Comment:
That's a big nerf, because then charcoal becomes less useful if you need to purge drugs, pyros, or anything not marked as toxin. 

Instead he needs to make the chemical come in a larger dose and shit.